error while installing GDAL 3.7.1 - webpdataset.cpp:564:22: error: ‘WebPBitstreamFeatures’ has no member named ‘format’

## Steps to reproduce the problem.

receiving the below error when running 'make' command
[ 22%] Built target gdal_WMS
[ 22%] Building CXX object frmts/webp/CMakeFiles/gdal_WEBP.dir/webpdataset.cpp.o
/code/GDAL/gdal-3.7.1/frmts/webp/webpdataset.cpp: In static member function ‘static GDALPamDataset* WEBPDataset::OpenPAM(GDALOpenInfo*)’:
/code/GDAL/gdal-3.7.1/frmts/webp/webpdataset.cpp:564:22: error: ‘WebPBitstreamFeatures’ has no member named ‘format’
         config.input.format == 2 ? "LOSSLESS" : "LOSSY", "IMAGE_STRUCTURE");
                      ^
make[2]: *** [frmts/webp/CMakeFiles/gdal_WEBP.dir/webpdataset.cpp.o] Error 1
make[1]: *** [frmts/webp/CMakeFiles/gdal_WEBP.dir/all] Error 2
make: *** [all] Error 2


## Operating system

Centos7

## GDAL version and provenance

GDAL version 3.7.1

--- a/frmts/webp/webpdataset.cpp
+++ b/frmts/webp/webpdataset.cpp
@@ -559,9 +559,13 @@ GDALPamDataset *WEBPDataset::OpenPAM(GDALOpenInfo *poOpenInfo)
         WebPGetFeatures(poOpenInfo->pabyHeader, poOpenInfo->nHeaderBytes,
                         &config.input) == VP8_STATUS_OK;
 
+    // Cf commit https://github.com/webmproject/libwebp/commit/86c0031eb2c24f78d4dcfc5dab752ebc9f511607#diff-859d219dccb3163cc11cd538effed461ff0145135070abfe70bd263f16408023
+    // Added in webp 0.4.0
+#if WEBP_DECODER_ABI_VERSION >= 0x0202
     poDS->GDALDataset::SetMetadataItem(
         "COMPRESSION_REVERSIBILITY",
         config.input.format == 2 ? "LOSSLESS" : "LOSSY", "IMAGE_STRUCTURE");
+#endif
 
     if (config.input.has_alpha)
         nBands = 4;

